src/python/pants/rules/core/cloc_test.py

```diff
@@ -57,6 +64,16 @@ def test_ignored(self) -> None:
     assert "Ignored the following files:" in output
     assert "empty.py: zero sized file" in output
 
+  def test_files_without_owners(self) -> None:
```

Comment:
Can you add a test which shows that:
Given:
`some/target:target` contains files `file1` and `file2`
`./pants cloc some/target/file1` doesn't include `file2`, which I _think_ is the expected behaviour as per https://docs.google.com/document/d/15xphZcFnowytF0Qu2sO1PG7wHAWoyLof7hK0jVa5T8E/edit?disco=AAAAI4KUmsI

Reply:
That is the expected behavior, yes, so a test is a good idea.